packages/eslint-plugin/docs/rules/no-use-before-define.md

```diff
@@ -67,7 +67,7 @@ let myVar: StringOrNumber;
 
 ```json
 {
-  "no-use-before-define": ["error", { "functions": true, "classes": true }]
+  "@typescript-eslint/no-use-before-define": ["error", { "functions": true, "classes": true }]
```

Comment:
Please add this line, as it's what we've standardised on for newer extension rules

```suggestion
  "no-use-before-define": "off",
  "@typescript-eslint/no-use-before-define": ["error", { "functions": true, "classes": true }]
```